frontend/src/utils/fake-backend.js

```diff
@@ -15,6 +15,8 @@ function configureFakeBackend() {
         switch (true) {
           case url.endsWith('/products/add') && method === 'POST':
```

Comment:
It might serve you well to move the method strings to and the paths into their own respective constants

Reply:
Hey @aaronpaulroberts, I'll add that in and submit a PR.